bedrock/press/forms.py

```diff
@@ -15,6 +15,70 @@
 SPEAKER_REQUEST_FILE_SIZE_LIMIT = 5242880  # 5MB
 
 
+class PressInquiryForm(forms.Form):
+    jobtitle = forms.CharField(
+        required=True,
+        widget=forms.TextInput(
+            attrs={
+                'class': 'required',
+                'required': 'required',
+                'aria-required': 'true'
+            }
+        )
+    )
+    name = forms.CharField(
+        required=True,
+        widget=forms.TextInput(
+            attrs={
+                'class': 'required',
+                'required': 'required',
+                'aria-required': 'true'
+            }
+        )
+    )
+    media_org = forms.CharField(
+        required=True,
+        widget=forms.TextInput(
+            attrs={
+                'class': 'required',
+                'required': 'required',
+                'aria-required': 'true'
+            }
+        )
+    )
+    inquiry = forms.CharField(
+        required=True,
+        widget=forms.Textarea(
+            attrs={
+                'class': 'required',
+                'required': 'required',
+                'aria-required': 'true'
+            }
+        )
+    )
+    deadline = forms.CharField(
+        required=True,
+        widget=forms.DateInput(
```

Comment:
Could get a `type="date"` rendered if you use our custom `DateInput` widget here. (Just `widget=DateInput` instead of `widget=forms.DateInput`.)